react-native.config.js

```diff
@@ -25,7 +25,14 @@ let config = {
     },
     android:{
       sourceDir: 'example/android',
-    }
+    },
+    windows:{
+      sourceDir: 'example\\windows',
+      solutionFile: 'NetInfoExample.sln',
+      project: {
+          projectFile: 'NetInfoExample\\NetInfoExample.vcxproj',
```

Comment:
@jonthysell 
I am not sure if this has been fixed in newer versions of react native. But the usage of \\ here could be problematic for ios users in other indirect scenarios. 

See my other PR where we ran into similar issues, we ended up having to patch for a while. 
https://github.com/wonday/react-native-orientation-locker/pull/188

Reply:
I had to patch this immediately it was painful here https://github.com/react-native-netinfo/react-native-netinfo/pull/522/commits/c9a2966e26a06bc887c5048742d4531d899732ad